theme/src/resources/templates/category_pages/liferay_dxp_cloud.hbs

```diff
@@ -19,6 +19,17 @@
 		</div>
 
 		<div class="col-md-8 offset-md-2" id="dxpCloudProductPage"></div>
+
+		<div class="col-md-12">
+			<div class="cta">
+				<h4>{{dc 'dxp_cloud-announcements_banner_description'}}</h4>
+
+				<a class="btn btn-outline-primary btn-sm"
+					href="https://discover.liferay.com/subscribe-to-liferay-dxp-cloud-updates">
```

Comment:
Do we really need to break this up onto 2 separate lines?  Can't we keep it on the same line?

Reply:
Prettier broke it up, I suppose it's exceeding 80char limit?